src/configExtractor.ts

```diff
@@ -1,16 +1,16 @@
-import { Codacyrc } from "codacy-seed"
+import {Codacyrc} from "codacy-seed"
 import * as glob from "glob"
 
-import { toolName } from "./toolMetadata"
+import {toolName} from "./toolMetadata"
 
-export async function extractPatternIdsToApply(
+export async function extractPatternsFromRC(
```

Comment:
It's hard to understand what `RC` means in my opinion.
Consider something like:
```suggestion
export async function extractPatternsFromCodacyrc(
```
which is also consistent with how we call the `codacyrc` file in other tools.